src/web/components/MostViewed/MostViewedFooter/MostViewedFooterData.tsx

```diff
@@ -50,8 +50,8 @@ function buildSectionUrl(
 	return joinUrl([ajaxUrl, `${endpoint}?dcr=true`]);
 }
 
-function buildDeeplyReadUrl(ajaxUrl: string, pillar: CAPIPillar) {
-	return joinUrl([ajaxUrl, 'most-read-deeply-read', `${pillar}.json?dcr`]);
+function buildDeeplyReadUrl(ajaxUrl: string) {
+	return joinUrl([ajaxUrl, 'most-read-deeply-read.json']);
```

Comment:
We're also removing `?dcr` - is that intentional?

Reply:
Yeah, that should only be needed on endpoints that are used on frontend and DCR, this is just DCR